src/csharp/Microsoft.Spark/Utils/PythonSerDe.cs

```diff
@@ -67,5 +67,31 @@ internal static object[] GetUnpickledObjects(Stream stream, int messageLength)
                 ArrayPool<byte>.Shared.Return(buffer);
             }
         }
+
+        /// <summary>
+        /// Custom pickler for GenericRow objects.
+        /// </summary>
+        internal class GenericRowPickler : IObjectPickler
```

Comment:
Is this the best place to put this class definition ?

Reply:
I thought `PythonSerDe.cs` would be doing pickling and unpickling. Where would you suggest to put this class?

Reply:
Why not put it in a file called `GenericRowPickler.cs`

Reply:
Sounds good. Added it to `GenericRowPickler.cs`.